swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -31,352 +26,137 @@ The following Azure Functions triggers and bindings are currently supported in B
 
-The Azure Functions functionality is implemented as a compiler extension. Thus, artifact generation happens automatically when you build a Ballerina module. Let's see how this works by building the above code. 
+## Create the app
 
-```
-$ bal build --cloud="azure_functions"
-Compiling source
-        wso2/azure_functions_deployment:0.1.0
-
-Generating executable
-        @azure_functions:Function: get-hello
-
-        Execute the command below to deploy the function locally.
-        func start --script-root target/azure_functions --java
-
-        Execute the command below to deploy Ballerina Azure Functions.
-        func azure functionapp publish <function_app_name> --script-root target/azure_functions 
-
-        target/bin/azure_functions_deployment.jar
-```
+Follow the steps below to create the Azure function app.
 
-### Deploy the function
-
-The created function app name should be provided to the placeholders shown in the above-generated usage instructions from the compiler. 
-
-A sample execution to deploy the functions to Azure Functions is shown below. 
-
-```bash
-$ func azure functionapp publish <function_app_name> --script-root target/azure_functions 
-Getting site publishing info...
-Creating archive for current directory...
-Uploading 28.64 MB [##############################################################################]
-Upload completed successfully.
-Deployment completed successfully.
-Syncing triggers...
-Functions in bal-learn-1:
-    get-hello - [httpTrigger]
-        Invoke url: https://bal-learn-1.azurewebsites.net/hello
-```
-
-### Invoke the function
-
-The deployed Azure Function can be tested by invoking it using an HTTP client such as cURL:
-
-```
-$ curl https://<function_app_name>.azurewebsites.net/hello\?name\=Jack
-Hello, Jack!
-```
-
-## More samples
-
-This section uses different types of triggers and bindings to build Azure functions to integrate with different Azure services using concepts explained in the above sections.
-
-### HTTP Trigger -> queue output
+1. Log in to the Azure CLI by executing the `az login` command on CLI.
+2. Create an <a href="https://docs.microsoft.com/en-us/azure/azure-functions/functions-create-function-app-portal" target="_blank">Azure Function app</a> with the given resource group with the following requirements.
+>**Note:** Make sure to remember the function application name and storage account name as they will be required in the code samples.
+   - Runtime stack - `Java 11`
+   - Hosting operating system - `Windows` (Currently, Linux is not supported in Azure by default for custom handlers.)
 
-The following Ballerina code gives an example of using an HTTP trigger to invoke the function and uses multiple output bindings to return the HTTP response and queue output binding to write an entry to a queue.
+## Write the function
```

Comment:
Shall we rename the title to something like sample code? as we are only showing a generic example here?

Reply:
This is applicable for aws lambda page as well

Reply:
The title `Write the function` connects well with the other topics in the flow of the use case.

Reply:
@anupama-pathirage - WDYT?

Reply:
Kept the title and added an intro sentence.